v0/destinations/zendesk/transform.js

```diff
@@ -558,7 +558,6 @@ async function processTrack(message, destinationConfig, headers) {
     userEmail = email;
   }
   zendeskUserID = zendeskUserID || userResponse.data.users[0].id;
-  userEmail = userEmail || userResponse.data.users[0].email;
```

Comment:
If the condition in line #545 is satisfied the `userEmail` is not going to get set right?

Reply:
the `userEmail` is already set and has a validation at lines 535 and 537 respectively. and inside the if condition there is an other check at line 554. So the above line (removed 561 line) is redundant

Reply:
let me know if I'm wrong

Reply:
Got it